src/SqlBackupTools/Program.cs

```diff
@@ -99,13 +100,20 @@ private static Logger CreateLogger(GeneralCommandInfos commandInfos)
             };
 
             var level = commandInfos.Verbose ? LogEventLevel.Debug : LogEventLevel.Information;
-            var loggerConf = new LoggerConfiguration()
-                .WriteTo.Async(conf =>
-                {
-                    var path = commandInfos.LogsPath?.Exists == true ? Path.Combine(commandInfos.LogsPath.FullName, logFileName) : "logs/" + logFileName;
-                    conf.File(path, rollingInterval: RollingInterval.Day, retainedFileCountLimit: 31, formatProvider: CultureInfo.InvariantCulture);
-                    conf.Console(formatProvider: CultureInfo.InvariantCulture);
-                }).MinimumLevel.Is(level);
+
+
+            var loggerConf = new LoggerConfiguration();
+
+            loggerConf.MinimumLevel.Is(level);
+
+            var sink = LoggerSinkConfiguration.Wrap(sink => new BackgroundWorkerSink(sink), conf =>
+            {
+                var path = commandInfos.LogsPath?.Exists == true ? Path.Combine(commandInfos.LogsPath.FullName, logFileName) : "logs/" + logFileName;
```

Comment:
Détail que tu ignorer:
```suggestion
                var basePath = commandInfos.LogsPath?.Exists == true ? commandInfos.LogsPath.FullName : "logs";
                var path = Path.Combine(basePath, logFileName);
```